corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -263,7 +263,7 @@
                 <div class="accordion-body collapse 
                     {% if cases|length == 1 %}in{% endif %}" 
                     id="form-case-acc-{{ forloop.counter }}">
-                    <div class="accordion-inner">
+                    <div class="accordion-inner" style="display: inline-block">
```

Comment:
If the only way to fix it was to style this particular node, yeah, I'd either add an id to this node or use a rule like `#form-case-data .accordion-inner`  It's not a huge advantage over the inline style, but having all styling information in one place saves a bit of time.